tests/fps_tests.py

```diff
@@ -500,16 +498,50 @@ def test_multi_invalid_etlds(self):
         loaded_sets = fp.load_sets()
         fp.find_invalid_eTLD_Plus1(loaded_sets)
         self.assertEqual(fp.error_list, 
-         ["The provided primary site does not have an eTLD in the" +
-                    " Public suffix list: https://primary.c2om",
-          "The provided alias does not have an eTLD in the" +
-                    " Public suffix list: https://primary.c2om",
-          "The provided aliased site does not have an eTLD in the" +
-                    " Public suffix list: https://primary.c2om",
-          "The provided associated site does not have an eTLD in the" +
-                    " Public suffix list: https://associated1.c2om",
-          "The provided service site does not have an eTLD in the" +
-                    " Public suffix list: https://service1.c2om"])
+         ["The provided primary site is not an eTLD+1: https://primary.c2om",
+          "The provided alias is not an eTLD+1: https://primary.c2om",
+          "The provided aliased site is not an eTLD+1: https://primary.c2om",
+          "The provided associated site is not an eTLD+1: https://associated1.c2om",
+          "The provided service site is not an eTLD+1: https://service1.c2om"])
+    def test_not_etld_plus1(self):
+        json_dict = {
+            "sets":
+            [
+                {
+                    "primary": "https://subdomain.primary.com",
+                    "ccTLDs": {},
+                    "rationaleBySite": {}
+                }
+            ]
+        }
+        fp = FpsCheck(fps_sites=json_dict,
+                     etlds=PublicSuffixList(
+                        psl_file = 'effective_tld_names.dat'),
+                     icanns=set())
+        loaded_sets = fp.load_sets()
+        fp.find_invalid_eTLD_Plus1(loaded_sets)
+        self.assertEqual(fp.error_list, 
+                         ["The provided primary site is not an eTLD+1: https://subdomain.primary.com"])
+        
+    def test_valid_etld_plus1(self):
+        json_dict = {
+            "sets":
+            [
+                {
+                    "primary": "https://primary.com.ar",
+                    "ccTLDs": {},
+                    "rationaleBySite": {}
+                }
+            ]
+        }
+        fp = FpsCheck(fps_sites=json_dict,
+                     etlds=PublicSuffixList(
+                        psl_file = 'effective_tld_names.dat'),
+                     icanns=set())
+        loaded_sets = fp.load_sets()
+        fp.find_invalid_eTLD_Plus1(loaded_sets)
+        self.assertEqual(fp.error_list, 
+                         [])
         
```

Comment:
Can we have a test case that checks whether a eTLD is an eTLD+1? E.g. trying to submit a set that includes `https://com` or `https://github.io` or something like that.

Reply:
Added test case "test_valid_tld_plus1" for this

Reply:
I've changed the test, but I had to change the check as well to meet the condition. As it was, the check would work with the `https://com` case but not the `https://github.io` case because of how `get_public_suffix` works. The current version should catch both cases (but I had to do some string manipulation with the "https://" prefix).